File: solutions/uri/2764/2764.py
import sys

for line in sys.stdin:
    day, month, year = map(int, line.split("/"))

    print(f"{month:02d}/{day:02d}/{year:02d}")
    print(f"{year:02d}/{month:02d}/{day:02d}")
    print(f"{day:02d}-{month:02d}-{year:02d}")

File: solutions/uri/2764/2764.cpp
#include <cstdio>

int main() {
    int day, month, year;

    while (scanf("%d/%d/%d", &day, &month, &year) != EOF) {
        printf("%02d/%02d/%02d\n", month, day, year);
        printf("%02d/%02d/%02d\n", year, month, day);
        printf("%02d-%02d-%02d\n", day, month, year);
    }

    return 0;
}
